src/main/java/org/fcrepo/importexport/common/BagProfile.java

```diff
@@ -111,15 +151,67 @@ public BagProfile(final InputStream in) throws IOException {
      * @return A map of field names to a Set of acceptable values (or null when the values are restricted).
      */
     public Map<String, Set<String>> getMetadataFields() {
-        return metadataFields;
+        return getMetadataFields(BAG_INFO_FIELDNAME);
     }
 
     /**
-     * Get the required APTrust-Info metadata fields.
-     * @return A map of field names to a Set of acceptable values (or null when the values are restricted),
-     *    or null when no APTrust-Info fields are required.
+     * Get the required tags for the extra tag file
+     *
+     * @param tagFile the tag file to get tags for
+     * @return map of tag = set of acceptable values, or null if tagFile doesn't exist
      */
-    public Map<String, Set<String>> getAPTrustFields() {
-        return aptrustFields;
+    public Map<String, Set<String>> getMetadataFields(final String tagFile) {
+        if (sections.contains(tagFile)) {
+            return metadataFields.get(tagFile);
+        }
+        return null;
+    }
+
+    /**
+     * Does a specific tag section exist in Other-Info
+     *
+     * @param section section to look for
+     * @return true if tagFile exists
+     */
+    public boolean hasSection(final String section) {
+        return (!sections.isEmpty() && sections.contains(section));
+    }
+
+    /**
+     * Does the profile have other tag files in Other-Info
+     *
+     * @return true if any extra tag files exist
+     */
+    public boolean hasExtraSections() {
```

Comment:
Same as above — this is only used for testing, can't we just use getSectionNames()?